mplib/examples/collision_avoidance.py

```diff
@@ -62,7 +62,7 @@ def add_point_cloud(self):
         box = trimesh.creation.box([0.1, 0.4, 0.2])
         points, _ = trimesh.sample.sample_surface(box, 1000)
         points += [0.55, 0, 0.1]
-        self.planner.update_point_cloud(points, radius=0.02)
+        self.planner.update_point_cloud(points, resolution=0.02)
         # add_point_cloud ankor end
 
     def demo(self, with_screw=True, use_point_cloud=True, use_attach=True):
```

Comment:
I guess now we can remove these arguments

Reply:
Sorry what arguments? the `resolution` or `use_point_cloud`/`use_attach`?
For the latter, yeah we can remove them, I think I just update the `demo(...)` method without changing its behavior. But we can make it simpler for people to start from, given this is an example.